package.json

```diff
@@ -25,6 +41,10 @@
     "publish:patch": "aragon apm publish patch",
     "publish:minor": "aragon apm publish minor",
     "publish:major": "aragon apm publish major",
-    "versions": "aragon apm versions"
+    "versions": "aragon apm versions",
+    "lint": "eslint .; solium --dir ./contracts",
+    "lint:fix": "eslint . --fix; solium --dir ./contracts --fix",
+    "coverage": "SOLIDITY_COVERAGE=true npm run ganache-cli:test",
```

Comment:
Maybe it makes sense to just move over the `test` script to use `npm run ganache-cli:test` too 🤷‍♂️.

Reply:
Done!

Reply:
Sorry, I missunderstood you the first time. Yes, I think it makes sense, since new developers usually fail to run `aragon devchain` first before to run `npm test`.